buildconfig/dev-packages/deb/mantid-developer/ns-control

```diff
@@ -53,6 +53,7 @@ Depends: git,
   python-dateutil,
   python-sphinx-bootstrap-theme,
   python-matplotlib,
+  python-parameterized
```

Comment:
This package doesn't exist on Ubuntu 16.04

Reply:
I moved the two tests using `parameterized` to for loops with no dependencies.